student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom:, **parameters)
+    super(**parameters)
```

Comment:
- [ ] I am unable to create a student object due to some errors (see image). I would recommend you refactor the `#initialize` and `#super` parameters to something like so:

  ```
  def initialize(age, classroom, name = "Unknown", parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    ...
  end
  ```

  ![student-err](https://user-images.githubusercontent.com/36542986/179576757-102645f0-cb2d-490a-9263-3e32f4159cfe.jpg)